src/utils/types.ts

```diff
@@ -0,0 +1,38 @@
+export interface PlayerInfo {
```

Comment:
I think Player interfaces could have better names; we have three Players and it can be hard to distinguish them solely on name.

Reply:
Maybe we could name this `FirebasePlayerInfo`? Maybe name all the Firebase types with the `Firebase` prefix?

Reply:
I changed it, please check if you think the new version is acceptable.